code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1792,7 +1792,7 @@
 	hasinner = 1
 	color_src = HAIR
 
-/datum/sprite_accessory/ears/cat/slime
+/datum/sprite_accessory/ears/cat/slime //I would yeet these, but unfortunately, deleting these can mess up savefiles, so you're stuck with this :)
```

Comment:
You can remove these. im pretty sure. Ask mark.

Reply:
Yeah, it'll just sanitize it and re-randomize if needed I believe.